<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUserDialog</class>
 <widget class="QDialog" name="AddUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add user</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>450</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="createUserLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>167</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create user:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>381</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="firstNameLabel">
        <property name="text">
         <string>First Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lastNameLabel">
        <property name="text">
         <string>Last Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="genderLabel">
        <property name="text">
         <string>Gender:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dateLabel">
        <property name="text">
         <string>Date of Birth:           </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="addressLabel">
        <property name="text">
         <string>Address:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cityLabel">
        <property name="text">
         <string>City:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="stateLabel">
        <property name="text">
         <string>State:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="postCodeLabel">
        <property name="text">
         <string>Post Code:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="homePhoneLabel">
        <property name="text">
         <string>Home phone:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mobilePhoneLabel">
        <property name="text">
         <string>Mobile phone:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="firstNameLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lastNameLineEdit"/>
      </item>
      <item>
       <widget class="QComboBox" name="genderComboBox">
        <item>
         <property name="text">
          <string>Female</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Male</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit">
        <property name="displayFormat">
         <string>dd.MM.yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="addressLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="cityLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="stateLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="postCodeLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="homePhoneLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="mobilePhoneLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddUserDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddUserDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
